src/main/groovy/io/openliberty/tools/gradle/tasks/AbstractTask.groovy

```diff
@@ -107,6 +100,22 @@ abstract class AbstractTask extends DefaultTask {
         return (installDir.exists() && new File(installDir, "lib/ws-launch.jar").exists())
     }
 
+    @Internal
+    protected boolean isLibertyInstalledAndValid(Project project) {
+        // only used for installLibertyTask, where it should throw an exception instead of returning false when the installDir is invalid.
+        if(project.liberty.installDir != null) {
+            project.getLogger().info("Found user-specified installDir. Checking if valid " + project.liberty.installDir)
```

Comment:
I think this log line should be removed. The call to checkAndAppendWlp method will potentially give a warning, and if no Liberty installation is found in this method we send an error.